docs/getting_started_guide/Installing_SIMP_using_r10k.rst

```diff
@@ -0,0 +1,263 @@
+Installing SIMP using r10k or Code Manager
+==========================================
+
+.. contents:: Contents:
+  :local:
+
+:term:`r10k` and :term:`Code Manager` are products that automate the development
+and deployment of a :term:`Puppet` infrastructure. SIMP supports the usage of these
+tools, with a little tweaking.
+
+Read the introduction documentation on whichever of these technologies that is
+being used:
+
+  * Code Manager: https://docs.puppet.com/pe/latest/code_mgr.html
+  * r10k: https://github.com/puppetlabs/r10k/blob/master/README.mkd
+
+.. NOTE::
+  r10k will be used to reference both r10k itself and Code Manager throughout
```

Comment:
These items need to be left aligned with the first character of the header. **Sometimes** it works, sometimes it doesn't, but aligning always works.